projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/ExecutionDataFetcher.java

```diff
@@ -76,19 +75,19 @@
 
 
    /**
-    * Currently, it does not provide filtering by specifying fields due
-    * to its post-pagination loading - the execution data is attached after slicing of the requested page.
+    * Populates the executions of the specified data jobs, that match the specified criteria.
     */
    List<V2DataJob> populateExecutions(List<V2DataJob> allDataJob, DataFetchingEnvironment dataFetchingEnvironment) {
-      final ExecutionQueryVariables queryVariables = fetchQueryVariables(dataFetchingEnvironment);
-      final Pageable pageable = constructPageable(queryVariables);
+      final Map<String, Object> arguments = dataFetchingEnvironment.getSelectionSet().getField(JobFieldStrategyBy.DEPLOYMENT_EXECUTIONS.getPath()).getArguments();
+      final DataJobExecutionQueryVariables dataJobExecutionQueryVariables = fetchDataJobExecutionQueryVariables(arguments);
```

Comment:
I think we need validation of _jobNameIn_ property. This field must be disabled in context of jobs query and we must send an error to the client. In context of particular job it does not make sense to filter by another one.

Reply:
I will add this validation.